packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -252,12 +259,27 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
             <li class={classes} id={this.guid} onClick={this.itemClickHandler}>
               {this.renderSelectIndicator(showDot, iconPath)}
               {this.renderIcon(iconPath)}
-              <span class="title">{this.textLabel}</span>
+              <span class="title">{this.renderTextContent()}</span>
             </li>
             {this.renderChildren()}
           </div>
         </InteractiveContainer>
       </Host>
     );
   }
+
+  private renderTextContent(): string | (string | VNode)[] {
+    if (!this.filterTextMatchPattern) {
+      return this.textLabel;
+    }
+
+    const parts: (string | VNode)[] = this.textLabel.split(this.filterTextMatchPattern);
+
+    if (parts.length > 1) {
+      // we only highlight the first match
+      parts[1] = <span class={CSS.filterMatch}>{parts[1]}</span>;
```

Comment:
fixme: should we use `strong` instead of `span` for semantics?

Reply:
Isn't the highlight purely visual and not important content? Not sure if `strong` is applicable in this case. 

From [MDN](http://127.0.0.1:63228/Dash/utxknrwq/developer.mozilla.org/en-US/docs/Web/HTML/Element/strong.html#usage_notes):

> The \<strong\> element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings). This could be a sentence that is of great importance to the whole page, or you could merely try to point out that some words are of greater importance compared to nearby content.

In the `<b>` vs `<strong>` section, it recommends using CSS for styling too:

> It may help to realize that both are valid and semantic elements in HTML and that it's a coincidence that they both have the same default styling (boldface) in most browsers (although some older browsers actually underline \<strong\>). Each element is meant to be used in certain types of scenarios, and if you want to bold text for decoration, you should instead actually use the CSS [font-weight](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight) property.

What do you think we should do? Would `<b>` would be more appropriate or should we keep the current approach?

Reply:
[`<mark>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/mark) is another option too.

Reply:
Yeah mark 👌🏼